rest_framework/schemas.py

```diff
@@ -531,7 +531,7 @@ def get_path_fields(self, path, method, view):
                 try:
                     model_field = model._meta.get_field(variable)
                 except:
-                    pass
+                    model_field = None  # Set model_field to None so it doesn't fail test
```

Comment:
Great, yup! We don't need the comment here, so drop that and this'll be good to go.